.probo.yaml

```
@@ -52,8 +55,32 @@ steps:
       - chown www-data:www-data -R /var/www/html/sites/simpletest
       - mkdir -p /var/www/html/sites/default/files/simpletest
       - chown www-data:www-data -R /var/www/html/sites/default/files/simpletest
-      - source $ASSET_DIR/credentials.sh
+      - source $ASSET_DIR/test-credentials.sh
       - export SIMPLETEST_BASE_URL="http://localhost"
-      - export SIMPLETEST_DB="mysql://${DATABASE_USER}:${DATABASE_PASS}@localhost/${DATABASE_NAME}"
+      - export SIMPLETEST_DB="mysql://${TEST_DATABASE_USER}:${TEST_DATABASE_PASS}@localhost/${TEST_DATABASE_NAME}"
       - cd /var/www/html/profiles/custom/az_quickstart
       - sudo -u www-data -E find $SRC_DIR/az-quickstart-scaffolding/web/profiles/custom/az_quickstart/ -name "*Test.php" | sudo -u www-data -E $SRC_DIR/az-quickstart-scaffolding/vendor/bin/fastest -vvv "$SRC_DIR/az-quickstart-scaffolding/vendor/bin/phpunit {} --colors="always" --verbose;"
+  - name: Log in to terminus
+    command: 'source ${ASSET_DIR}/terminus.sh; terminus auth:login --machine-token=${TERMINUS_TOKEN}'
+  # - name: Set up migration data
+  #   command: 'source ${ASSET_DIR}/migration.sh'
+  - name: Migration Tests
+    plugin: Script
+    script:
+      - source ${ASSET_DIR}/migration.sh
+      - terminus connection:info ${MIGRATION_SOURCE_SITE_NAME}.dev --fields 'mysql_host,mysql_password,mysql_port,mysql_database,mysql_username' --format=json > /tmp/migration_config.json
+      - mkdir -p /var/www/html/sites/migration && cd /var/www/html/sites
+      - cp example.sites.php sites.php
+      - echo "\$sites['$BUILD_ID--site-migration.probo.build'] = 'migration';" >> sites.php
+      - cp ${ASSET_DIR}/migration.settings.php migration/settings.php
+      - cd migration
+      - mkdir files && chown www-data:www-data -R ./files
+      - drush site:install az_quickstart --site-name='Quickstart Migration' --account-name=azadmin --account-pass=azadmin --verbose --yes -n
+      - terminus env:wake ${MIGRATION_SOURCE_SITE_NAME}.dev
+      - drush user:role:add az_content_admin azcontentadmin
+      - drush -y en az_migration
+      - drush -y cset az_migration.settings migrate_d7_protocol "https"
+      - drush -y cset az_migration.settings migrate_d7_filebasepath "dev-${MIGRATION_SOURCE_SITE_NAME}.pantheonsite.io/"
+      - drush -y cset az_migration.settings migrate_d7_public_path "sites/default/files"
+      - drush mim --group=az_migration
```

Let's move the static user account creation steps here:

```suggestion
      - drush mim --group=az_migration
      - drush user:create azadmin --password="azadmin"
      - drush user:role:add administrator azadmin
```